pkg/cluster/manager/deploy.go

```diff
@@ -245,6 +245,10 @@ func (m *Manager) Deploy(
 		deployDir := spec.Abs(globalOptions.User, inst.DeployDir())
 		// data dir would be empty for components which don't need it
 		dataDirs := spec.MultiDirAbs(globalOptions.User, inst.DataDir())
+
+		log.Debugf("inst.ComponentName=%+v, inst.DataDir=%+v, dataDirs=%+v\n",
```

Comment:
Did you test how it behaviors when using in real world, would it broke the dynamic TUI output?